pkg/scalers/postgresql_scaler.go

```diff
@@ -4,14 +4,14 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	_ "github.com/lib/pq"
```

Comment:
@kashifsoofi I think this import should stay as it was _ "github.com/lib/pq" as this is a required PostgreSQL drive.

Reply:
@turbaszek sure will add that back then

Reply:
Thanks a lot!

Reply:
I am new to go, is there a way to ignore go vet for this import/file?

Reply:
Hm, I checkouted your PR and it seems that running `make govet` finds no issues if the `pq` import is present

Reply:
Last commit is removing it, after restoring it, it is complaining :(

Reply:
let me commit and see if it works on CI

Reply:
I think it was the order of imports no more error after moving `strconv` after `fmt`

Reply:
Failing on static check now, any pointers

Reply:
I'm currently out but I will take a look once I'm back

Reply:
@kashifsoofi I think  that adding a comment on line before the import should do the trick:
```
// PostreSQL drive required for this scaler
_ "github.com/lib/pq"
```
```
pkg/scalers/postgresql_scaler.go:9:2: a blank import should be only in a main or test package, or have a comment justifying it
Found 1 lint suggestions; failing.
```

Reply:
Also it's possible to run the statick checks locally using [precommit](https://github.com/kedacore/keda/blob/v2/CONTRIBUTING.md#code-quality)

Reply:
Thanks @turbaszek